### DISCUSSION TITLE
Will Tiddlywiki have a blank or user type for tiddlers?

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6178] [Q&A] [answered]
I have seen some activity on viewtemplate and pagetemplate recently!

With the new features in TW 5.1.23+ it gets easier to get a tiddler text field and parse it and display in some different form! example is Section Editor!

My question is: Will TW have a blank or user type for tiddlers, lets user to decide how to display tiddlers in story river?
That means by default nothing will be shown and user have flexibility to decide how, what and where display tiddler fields/content in a customized order!

**Comment by pmario** [comment, marked as answer]
That's what is built with the PR #6168: **Add "cascade" filter run prefix and use it to control view templates** .. Once that's finished, you can forget all about the "hide-body" complexity. It will be much more flexible, once you understand how it works now. 

The mechanism we make "custom" ViewTemplates now will be obsolete. You can pretty much use any field that fit's your needs to activate your ViewTemplate on a per tiddler basis.

**Comment by kookma** [comment]
Hi @pmario 
This will be amazing! and can bring many new opportunities!

consider on can render a tiddler in story river with without displaying title. This is what is used in Fishing plugin! or prevent rendering text field like Section Editor, or create nice forms just by reading fields, ...

Wow I see endless possibilities!

**Reply by pmario** [reply]
Yea, that's exactly what will be possible ... no limits :)
